hybridse/src/udf/udf.cc

```diff
@@ -159,7 +159,13 @@ int32_t weekofyear(Date *date) {
 
 float Cotf(float x) { return cosf(x) / sinf(x); }
 
-double Degrees(double x) { return x * (180 / 3.141592653589793238463L); }
+double M_PI = 3.141592653589793238463L;
```

Comment:
`M_PI` is already a defined macro. cannot use the same variable.

Reply:
I‘ll remove the definition of this line.